libs/service-portal/information/src/lib/messages.ts

```diff
@@ -387,3 +387,59 @@ export const mCompany = defineMessages({
     defaultMessage: 'ÍSAT Atvinnugreinaflokkun',
   },
 })
+
+export const mNotifications = defineMessages({
+  intro: {
+    id: 'sp.notifications:intro-text',
```

Comment:
Just a heads up you should add this
`useNamespaces('sp.notifications')` within the screen that uses these strings. 😃

Reply:
Ahh nice to know, thanks 🙏